aten/src/ATen/test/test_install/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ find_package(ATen REQUIRED)
 include_directories(${ATEN_INCLUDE_DIR})
 
 # C++11
-set(CMAKE_CXX_FLAGS "--std=c++11 ${CMAKE_CXX_FLAGS}")
+if (not MSVC) 
+    set(CMAKE_CXX_FLAGS "--std=c++11 ${CMAKE_CXX_FLAGS}") 
+endif()
```

Comment:
I'm... surprised this ever worked at all. (Don't we have to still tell MSVC to use C++11? Maybe it's default??)

Reply:
MSVC doesn't actually have a compiler option to use C++11 (see https://blogs.msdn.microsoft.com/vcblog/2016/06/07/standards-version-switches-in-the-compiler/). Previously the compiler was just throwing a warning and ignoring the flag. Given that everything seems to work fine, I guess this doesn't break anything?

Reply:
SGTM